en-US/N.xml

```diff
@@ -111,17 +116,14 @@
 			<term>number sign</term>
 			 <listitem>
 				<para>
-					Generally, use "number sign" to refer to the # character. Alternatively, use "hash" to refer to a hashtag in social media, or to refer to Secure Hash Algorithm (see <xref linkend="sha" />), or when writing exclusively for a European audience. You can instead use "pound sign" when writing exclusively for a North American audience, if "number sign" is not appropriate for the context.  
+					Generally, use "number sign" to refer to the # character.
+          Or, use "hash" to refer to a hashtag in social media, or to refer to Secure Hash Algorithm (see <xref linkend="sha" />), or when writing exclusively for a European audience.
```

Comment:
I know that Microsoft advises otherwise, but I don't much like this word replacement at the start of this sentence, which results in a sentence in the format "Or A, or B, or C".